www/source/partials/docs/_dev-pkgs-write-plans.html.md.erb

```diff
@@ -54,22 +54,14 @@ All plans must have a `plan.sh` or `plan.ps1` at the root of the plan context. T
 
 3. The easiest way to create a plan is to use the `hab plan init` subcommand. This subcommand will create a directory, known as the plan context, that contains your plan file and any runtime hooks and/or templated configuration data.
 
-    To use `hab plan init` as part of your project repo, navigate to the root of your project repo and run `hab plan init`. It will create a new `habitat` sub-directory with a plan.sh based on the name of the parent directory, and include a `default.toml` file as well as `config` and `hooks` directories for you to populate as needed. For example:
+    To use `hab plan init` as part of your project repo, navigate to the root of your project repo and run `hab plan init`. It will create a new `habitat` sub-directory with a plan.sh (or plan.ps1 on Windows) based on the name of the parent directory, and include a `default.toml` file as well as `config` and `hooks` directories for you to populate as needed. For example:
 
     ```shell
     $ cd /path/to/<reponame>
     $ hab plan init
     ```
 
-    will result in a new `habitat` directory located at `/path/to/<reponame>/habitat`. A plan.sh file will be created and the `pkg_name` variable in plan.sh will be set to _\<reponame\>_. Also, any environment variables that you have previouly set (such as `HAB_ORIGIN`) will be used to populate the respective `pkg_*` variables.
-
-    If you want to generate a Powershell based plan file for building Windows packages, use the `--windows` option:
-
-    ```Powershell
-    PS C:\\> hab plan init --windows yourplan
-    ```
-
-    this generates a `plan.ps1` with proper Powershell variables, arrays, hashtables and functions.
+    will result in a new `habitat` directory located at `/path/to/<reponame>/habitat`. A plan file will be created and the `pkg_name` variable will be set to _\<reponame\>_. Also, any environment variables that you have previouly set (such as `HAB_ORIGIN`) will be used to populate the respective `pkg_*` variables.
```

Comment:
Should this also say `(or C:\path\to\<reponame>\habitat on windows)` ?

Reply:
same as above

Reply:
actually this one should remain as is because above on line 60 in the code example we have `$ cd /path/to/<reponame>`. This partial can be applied to either platform so we stay away from `c:\`.